addons/ui/org.openhab.ui.cometvisu/OSGI-INF/cv-backend-login.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.openhab.ui.cometvisu.backend.LoginResource">
+   <implementation class="org.openhab.ui.cometvisu.backend.LoginResource"/>
+   <service>
+      <provide interface="org.openhab.ui.cometvisu.backend.LoginResource"/>
```

Comment:
All your JAX-RS services should also register as a RESTResource (see e.g. https://github.com/eclipse/smarthome/blob/master/bundles/io/org.eclipse.smarthome.io.rest.core/OSGI-INF/bindingresource.xml#L15)